plugins/woocommerce/includes/legacy/class-wc-legacy-api.php

```diff
@@ -114,7 +114,12 @@ private function maybe_log_rest_api_request( string $route, ?string $user_agent
 		 * @since 8.5.0
 		 */
 		if ( apply_filters( 'woocommerce_log_legacy_rest_api_usages', true ) ) {
-			wc_get_logger()->info( 'Version: ' . WC_API_REQUEST_VERSION . ", Route: $route, User agent: $user_agent", array( 'source' => 'legacy_rest_api_usages' ) );
+            $info = 'Version: ' . WC_API_REQUEST_VERSION . ", Route: $route, User agent: $user_agent";
+            $ip_address = WC_Geolocation::get_ip_address();
+            if('' !== $ip_address) {
```

Comment:
The linter hasn't run yet, but I'd expect it to want you to something like this:

```suggestion
            if( '' !== $ip_address ) {
```